test/cli.js

```diff
@@ -0,0 +1,29 @@
+import test from 'ava';
+import sinon from 'sinon';
+
+import * as shxModule from '../src/shx';
+
+const sandbox = sinon.sandbox.create();
+const shx = sandbox.spy(shxModule, 'shx');
```

Comment:
Example use of named exports for spying where there was previously a default export.  The default export has no reference to maintain, whereas a wildcard import does.